Source package : fuelmanager
File name       : fuelmanager-0.5.1.tar.bz2
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/fuelmanager/fuelmanager-0.5.1.tar.bz2/sha512/a5d0655eac56004d79314ad425f4ad12b90ee2f4fd3b9d295481203ea9a7c9a1877a803bedea35627b422a59975f86e35ab2f9460b051b57569d3b82fabe4479/fuelmanager-0.5.1.tar.bz2
Source URL      : https://gitlab.com/kc8hfi/fuelmanager/-/archive/0.5.1/fuelmanager-0.5.1.tar.bz2

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-fuelmanager-0.5.1.tar.bz2: bzip2 compressed data, block size = 900k
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-fuelmanager-0.5.1.tar.bz2: bzip2 compressed data, block size = 900k


Only in lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1: build
Only in lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1: .directory
diff -r source-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1/fuelmanager.pro.user lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1/fuelmanager.pro.user
3c3
< <!-- Written by QtCreator 4.9.2, 2021-01-11T23:17:59. -->
---
> <!-- Written by QtCreator 4.14.1, 2022-04-03T22:52:47. -->
7c7
<   <value type="QByteArray">{9b78ec33-97fe-41ea-84a5-b76c6116e6f0}</value>
---
>   <value type="QByteArray">{7921d93a-08a8-423e-8583-7d66a794fe2a}</value>
51a52
>    <value type="QString" key="EditorConfiguration.ignoreFileTypes">*.md, *.MD, Makefile</value>
52a54
>    <value type="bool" key="EditorConfiguration.skipTrailingWhitespace">true</value>
57a60,69
>    <valuemap type="QVariantMap" key="AutoTest.ActiveFrameworks">
>     <value type="bool" key="AutoTest.Framework.Boost">true</value>
>     <value type="bool" key="AutoTest.Framework.Catch">true</value>
>     <value type="bool" key="AutoTest.Framework.GTest">true</value>
>     <value type="bool" key="AutoTest.Framework.QtQuickTest">true</value>
>     <value type="bool" key="AutoTest.Framework.QtTest">true</value>
>    </valuemap>
>    <valuemap type="QVariantMap" key="AutoTest.CheckStates"/>
>    <value type="int" key="AutoTest.RunAfterBuild">0</value>
>    <value type="bool" key="AutoTest.UseGlobal">true</value>
59a72,82
>    <value type="QString" key="ClangCodeModel.WarningConfigId">Builtin.Questionable</value>
>    <valuemap type="QVariantMap" key="ClangTools">
>     <value type="bool" key="ClangTools.AnalyzeOpenFiles">true</value>
>     <value type="bool" key="ClangTools.BuildBeforeAnalysis">true</value>
>     <value type="QString" key="ClangTools.DiagnosticConfig">Builtin.DefaultTidyAndClazy</value>
>     <value type="int" key="ClangTools.ParallelJobs">8</value>
>     <valuelist type="QVariantList" key="ClangTools.SelectedDirs"/>
>     <valuelist type="QVariantList" key="ClangTools.SelectedFiles"/>
>     <valuelist type="QVariantList" key="ClangTools.SuppressedDiagnostics"/>
>     <value type="bool" key="ClangTools.UseGlobalSettings">true</value>
>    </valuemap>
65c88
< <<<<<<< HEAD
---
>    <value type="QString" key="DeviceType">Desktop</value>
68,73c91
<    <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">{5761e3c4-57e3-4615-acd1-12817d52ed4b}</value>
< =======
<    <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Qt 5.11.3 (System)</value>
<    <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName">Qt 5.11.3 (System)</value>
<    <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">{7d169f61-3c2c-40bd-a34a-b16f6bbc8853}</value>
< >>>>>>> 551d9ac0a8f91922dbf8ae08587466266eb1470b
---
>    <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">{f8b32835-9b78-4678-b66e-b8e066570677}</value>
78,138c96,97
< <<<<<<< HEAD
<     <value type="QString" key="ProjectExplorer.BuildConfiguration.BuildDirectory">/home/kc8hfi/programming/qt/build-fuelmanager-Desktop-Debug</value>
<     <valuemap type="QVariantMap" key="ProjectExplorer.BuildConfiguration.BuildStepList.0">
<      <valuemap type="QVariantMap" key="ProjectExplorer.BuildStepList.Step.0">
<       <value type="bool" key="ProjectExplorer.BuildStep.Enabled">true</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">qmake</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">QtProjectManager.QMakeBuildStep</value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.LinkQmlDebuggingLibrary">true</value>
<       <value type="QString" key="QtProjectManager.QMakeBuildStep.QMakeArguments"></value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.QMakeForced">false</value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.SeparateDebugInfo">false</value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.UseQtQuickCompiler">false</value>
<      </valuemap>
<      <valuemap type="QVariantMap" key="ProjectExplorer.BuildStepList.Step.1">
<       <value type="bool" key="ProjectExplorer.BuildStep.Enabled">true</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Make</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">Qt4ProjectManager.MakeStep</value>
<       <valuelist type="QVariantList" key="Qt4ProjectManager.MakeStep.BuildTargets"/>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.Clean">false</value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeArguments"></value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeCommand"></value>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.OverrideMakeflags">false</value>
<      </valuemap>
<      <value type="int" key="ProjectExplorer.BuildStepList.StepsCount">2</value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Build</value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">ProjectExplorer.BuildSteps.Build</value>
<     </valuemap>
<     <valuemap type="QVariantMap" key="ProjectExplorer.BuildConfiguration.BuildStepList.1">
<      <valuemap type="QVariantMap" key="ProjectExplorer.BuildStepList.Step.0">
<       <value type="bool" key="ProjectExplorer.BuildStep.Enabled">true</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Make</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">Qt4ProjectManager.MakeStep</value>
<       <valuelist type="QVariantList" key="Qt4ProjectManager.MakeStep.BuildTargets"/>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.Clean">true</value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeArguments">clean</value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeCommand"></value>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.OverrideMakeflags">false</value>
<      </valuemap>
<      <value type="int" key="ProjectExplorer.BuildStepList.StepsCount">1</value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Clean</value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">ProjectExplorer.BuildSteps.Clean</value>
<     </valuemap>
<     <value type="int" key="ProjectExplorer.BuildConfiguration.BuildStepListCount">2</value>
<     <value type="bool" key="ProjectExplorer.BuildConfiguration.ClearSystemEnvironment">false</value>
<     <valuelist type="QVariantList" key="ProjectExplorer.BuildConfiguration.UserEnvironmentChanges"/>
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Debug</value>
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName">Debug</value>
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">Qt4ProjectManager.Qt4BuildConfiguration</value>
<     <value type="int" key="Qt4ProjectManager.Qt4BuildConfiguration.BuildConfiguration">2</value>
<     <value type="bool" key="Qt4ProjectManager.Qt4BuildConfiguration.UseShadowBuild">true</value>
<    </valuemap>
<    <valuemap type="QVariantMap" key="ProjectExplorer.Target.BuildConfiguration.1">
<     <value type="QString" key="ProjectExplorer.BuildConfiguration.BuildDirectory">/home/kc8hfi/programming/qt/build-fuelmanager-Desktop-Release</value>
< =======
<     <value type="QString" key="ProjectExplorer.BuildConfiguration.BuildDirectory">/home/kc8hfi/programming/qt/fuelmanager-0.5</value>
< >>>>>>> 551d9ac0a8f91922dbf8ae08587466266eb1470b
---
>     <value type="QString" key="ProjectExplorer.BuildConfiguration.BuildDirectory">/home/kc8hfi/programming/qt/fuelmanager</value>
>     <value type="QString" key="ProjectExplorer.BuildConfiguration.BuildDirectory.shadowDir">/home/kc8hfi/programming/qt/fuelmanager</value>
142,143d100
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">qmake</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
145d101
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.LinkQmlDebuggingLibrary">false</value>
148,149c104
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.SeparateDebugInfo">false</value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.UseQtQuickCompiler">false</value>
---
>       <valuelist type="QVariantList" key="QtProjectManager.QMakeBuildStep.SelectedAbis"/>
153,154d107
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Make</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
156,160d108
<       <valuelist type="QVariantList" key="Qt4ProjectManager.MakeStep.BuildTargets"/>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.Clean">false</value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeArguments"></value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeCommand"></value>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.OverrideMakeflags">false</value>
164c112
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
---
>      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName">Build</value>
170,171d117
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Make</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
173,174d118
<       <valuelist type="QVariantList" key="Qt4ProjectManager.MakeStep.BuildTargets"/>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.Clean">true</value>
176,177d119
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeCommand"></value>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.OverrideMakeflags">false</value>
181c123
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
---
>      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName">Clean</value>
185a128
>     <valuelist type="QVariantList" key="ProjectExplorer.BuildConfiguration.CustomParsers"/>
187d129
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Release</value>
191,248c133
<     <value type="bool" key="Qt4ProjectManager.Qt4BuildConfiguration.UseShadowBuild">true</value>
<    </valuemap>
< <<<<<<< HEAD
<    <valuemap type="QVariantMap" key="ProjectExplorer.Target.BuildConfiguration.2">
<     <value type="QString" key="ProjectExplorer.BuildConfiguration.BuildDirectory">/home/kc8hfi/programming/qt/build-fuelmanager-Desktop-Profile</value>
<     <valuemap type="QVariantMap" key="ProjectExplorer.BuildConfiguration.BuildStepList.0">
<      <valuemap type="QVariantMap" key="ProjectExplorer.BuildStepList.Step.0">
<       <value type="bool" key="ProjectExplorer.BuildStep.Enabled">true</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">qmake</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">QtProjectManager.QMakeBuildStep</value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.LinkQmlDebuggingLibrary">true</value>
<       <value type="QString" key="QtProjectManager.QMakeBuildStep.QMakeArguments"></value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.QMakeForced">false</value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.SeparateDebugInfo">true</value>
<       <value type="bool" key="QtProjectManager.QMakeBuildStep.UseQtQuickCompiler">true</value>
<      </valuemap>
<      <valuemap type="QVariantMap" key="ProjectExplorer.BuildStepList.Step.1">
<       <value type="bool" key="ProjectExplorer.BuildStep.Enabled">true</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Make</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">Qt4ProjectManager.MakeStep</value>
<       <valuelist type="QVariantList" key="Qt4ProjectManager.MakeStep.BuildTargets"/>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.Clean">false</value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeArguments"></value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeCommand"></value>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.OverrideMakeflags">false</value>
<      </valuemap>
<      <value type="int" key="ProjectExplorer.BuildStepList.StepsCount">2</value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Build</value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">ProjectExplorer.BuildSteps.Build</value>
<     </valuemap>
<     <valuemap type="QVariantMap" key="ProjectExplorer.BuildConfiguration.BuildStepList.1">
<      <valuemap type="QVariantMap" key="ProjectExplorer.BuildStepList.Step.0">
<       <value type="bool" key="ProjectExplorer.BuildStep.Enabled">true</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Make</value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<       <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">Qt4ProjectManager.MakeStep</value>
<       <valuelist type="QVariantList" key="Qt4ProjectManager.MakeStep.BuildTargets"/>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.Clean">true</value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeArguments">clean</value>
<       <value type="QString" key="Qt4ProjectManager.MakeStep.MakeCommand"></value>
<       <value type="bool" key="Qt4ProjectManager.MakeStep.OverrideMakeflags">false</value>
<      </valuemap>
<      <value type="int" key="ProjectExplorer.BuildStepList.StepsCount">1</value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Clean</value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">ProjectExplorer.BuildSteps.Clean</value>
<     </valuemap>
<     <value type="int" key="ProjectExplorer.BuildConfiguration.BuildStepListCount">2</value>
<     <value type="bool" key="ProjectExplorer.BuildConfiguration.ClearSystemEnvironment">false</value>
<     <valuelist type="QVariantList" key="ProjectExplorer.BuildConfiguration.UserEnvironmentChanges"/>
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Profile</value>
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName">Profile</value>
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">Qt4ProjectManager.Qt4BuildConfiguration</value>
<     <value type="int" key="Qt4ProjectManager.Qt4BuildConfiguration.BuildConfiguration">0</value>
<     <value type="bool" key="Qt4ProjectManager.Qt4BuildConfiguration.UseShadowBuild">true</value>
---
>     <value type="int" key="RunSystemFunction">0</value>
250,251d134
<    <value type="int" key="ProjectExplorer.Target.BuildConfigurationCount">3</value>
< =======
253d135
< >>>>>>> 551d9ac0a8f91922dbf8ae08587466266eb1470b
258c140
<      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
---
>      <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName">Deploy</value>
262,263c144,145
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Deploy Configuration</value>
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
---
>     <valuemap type="QVariantMap" key="ProjectExplorer.DeployConfiguration.CustomData"/>
>     <value type="bool" key="ProjectExplorer.DeployConfiguration.CustomDataEnabled">false</value>
267d148
<    <valuemap type="QVariantMap" key="ProjectExplorer.Target.PluginSettings"/>
274a156,163
>     <valuelist type="QVariantList" key="Analyzer.Perf.RecordArguments">
>      <value type="QString">-e</value>
>      <value type="QString">cpu-cycles</value>
>      <value type="QString">--call-graph</value>
>      <value type="QString">dwarf,4096</value>
>      <value type="QString">-F</value>
>      <value type="QString">250</value>
>     </valuelist>
317a207
>     <valuelist type="QVariantList" key="CustomOutputParsers"/>
320,322d209
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">fuelmanager</value>
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>
< <<<<<<< HEAD
324,333c211
<     <value type="QString" key="RunConfiguration.Arguments"></value>
< =======
<     <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">Qt4ProjectManager.Qt4RunConfiguration:/home/kc8hfi/programming/qt/fuelmanager-0.5/fuelmanager.pro</value>
<     <value type="bool" key="QmakeProjectManager.QmakeRunConfiguration.UseLibrarySearchPath">true</value>
<     <value type="QString" key="Qt4ProjectManager.Qt4RunConfiguration.CommandLineArguments"></value>
<     <value type="QString" key="Qt4ProjectManager.Qt4RunConfiguration.ProFile">fuelmanager.pro</value>
<     <value type="QString" key="Qt4ProjectManager.Qt4RunConfiguration.UserWorkingDirectory"></value>
<     <value type="QString" key="Qt4ProjectManager.Qt4RunConfiguration.UserWorkingDirectory.default">/home/kc8hfi/programming/qt/fuelmanager-0.5</value>
< >>>>>>> 551d9ac0a8f91922dbf8ae08587466266eb1470b
<     <value type="uint" key="RunConfiguration.QmlDebugServerPort">3768</value>
---
>     <value type="QString" key="ProjectExplorer.RunConfiguration.BuildKey">/home/kc8hfi/programming/qt/fuelmanager/fuelmanager.pro</value>
337d214
<     <value type="bool" key="RunConfiguration.UseMultiProcess">false</value>
340,341c217
<     <value type="QString" key="RunConfiguration.WorkingDirectory"></value>
<     <value type="QString" key="RunConfiguration.WorkingDirectory.default">/home/kc8hfi/programming/qt/build-fuelmanager-Desktop-Debug</value>
---
>     <value type="QString" key="RunConfiguration.WorkingDirectory.default">/home/kc8hfi/programming/qt/fuelmanager</value>
352c228
<   <value type="int">21</value>
---
>   <value type="int">22</value>
356c232
<   <value type="int">21</value>
---
>   <value type="int">22</value>
Only in lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1: fuelmanager.pro.user.9b78ec3.4.10-pre1
Only in lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1: .git
Only in lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1: history
Only in lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1/icons/256x256: .directory
Only in lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1/resources: .directory
Only in lookaside-fuelmanager-0.5.1.tar.bz2-extracted/fuelmanager-0.5.1: ui
